packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -137,3 +139,30 @@ export const makeDefaultTestContext = async (t, makeSpace) => {
     simpleCreatePriceFeed,
   };
 };
+
+/**
+ * @param {Awaited<ReturnType<typeof coalesceUpdates>>} state
+ * @param {Brand<'nat'>} brand
+ */
+export const purseBalance = (state, brand) => {
```

Comment:
passing `state` to a helper seems like an anti-pattern, since `state` is mutable. Can this be done by passing `state.balances` since the operation is supposed to be read-only?

Reply:
it could be but since this is a test I've opted for ease of writing and reading.

Is this a blocking request?

Reply:
Not if it's a lot of work. How much work does it save?

Reply:
My objection wasn't on amount of work to make the change. It was that it made calling require more typing.

In this case the function was no longer used so I've removed it.